angelsbioprocessing/locale/en/bio-processing.cfg

```diff
@@ -556,3 +578,50 @@ desert-garden-b=Advanced desert seed extraction
 
 swamp-garden-a=Swamp seed extraction
 swamp-garden-b=Advanced swamp seed extraction
+
+garden-cultivating=A low chance to grow gardens that you do not have yet. Not suitable for garden duplication.
+temperate-garden-cultivating-b=Duplicate a temperate garden. Suitable for perpetual generation of alien plant-life samples.
+desert-garden-cultivating-b=Duplicate a desert garden. Suitable for perpetual generation of alien plant-life samples.
+swamp-garden-cultivating-b=Duplicate a swamp garden. Suitable for perpetual generation of alien plant-life samples.
```

Comment:
Not so explicit, maybe something like use plant-life knowledge to grow more fertile gardens?

Reply:
As I said above, I will remove all recipe discriptions.